bookserver/config.py

```diff
@@ -81,11 +80,16 @@ def database_url(self) -> str:
     web2py_path: str = str(Path(__file__).parents[2] / "web2py")
 
     # This is the private key web2py uses for hashing passwords.
-    @property  # type: ignore
-    @lru_cache
+    @property
     def web2py_private_key(self) -> str:
-        with open(self.web2py_path, encoding="utf-8") as f:
-            return f.read()
+        authfile = Path(self.web2py_path) / "private" / "auth.key"
```

Comment:
Oops! Thanks for this fix.

Reply:
I inteded `webpy_path` to point to the root web2py folder, so I'm using a slightly different path (see previous comment).

Reply:
I can see that, but I am currently in a mixed environment where web2py is in a Docker container and BookServer is not.  So pointing to the parent of the private directory makes sense for me.

Reply:
I'll be using these assumptions about `web2py_path` in the test code -- let me know if that boogers something up for you.